LICENSE.md

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017, Imperial College, London
```

Comment:
Remove the commas (i.e. "Copyright (c) 2017 Imperial College London") to be consistent with the "author" field in `setup.py` and the structure of the copyright line given [here](https://opensource.org/licenses/MIT).